src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,29 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
+    this.health = health;
+    this.name = name;
+    this.hidden = hidden;
```

Comment:
The 'hidden' property is being initialized in the Animal class, but it should only be relevant for Herbivores. Consider removing it from here and initializing it in the Herbivore class instead.